build/DerivedHostMachineInfo.props

```diff
@@ -2,8 +2,7 @@
   <PropertyGroup>
     <IsDebianBaseDistro Condition=" '$(HostOSName)' == 'ubuntu' OR '$(HostOSName)' == 'debian' ">true</IsDebianBaseDistro>
     <IsRPMBasedDistro Condition=" $(HostRid.StartsWith('rhel')) AND '$(HostRid)' != 'rhel.6-x64' ">true</IsRPMBasedDistro>
-
-    <PublishNativeInstallers Condition=" '$(IslinuxPortable)' != 'true' AND '$(HostRid)' != 'rhel.6-x64' ">true</PublishNativeInstallers>
+    <PublishNativeInstallers Condition=" '$(IslinuxPortable)' != 'true' AND '$(HostRid)' != 'rhel.6-x64' AND '$(HostRid)' != 'alpine.3.6-x64'">true</PublishNativeInstallers>
```

Comment:
It seems like this property would be better to default to if we are creating native installers at all. If we aren't even creating native installers, then there is no reason to check the Rid.

So it seems that this property should be based on `$(SkipBuildingInstallers)`.  If `$(SkipBuildingInstallers) == true`, this property should be false.

Reply:
@johnbeisner made it this way few months ago. John, what do you think?